lib/formtools/renderers-cell.js

```diff
@@ -8,7 +8,7 @@ var dateRenderer = module.exports.date = function dateRenderer (date, record, fi
     return callback(null, moment(date).format(linz.get('date format')));
 };
 
-var dateTimeRenderer = module.exports.dateTime = function dateTimeRenderer (date, record, fieldName, model, callback) {
```

Comment:
I think we've discussed this, but why are we moving away from camel casing here?